pocs/observatory.py

```diff
@@ -186,8 +186,8 @@ def get_observation(self, *args, **kwargs):
         self.logger.debug("Getting observation for observatory")
 
         # If observation list is empty or a reread is requested
-        if self.scheduler.has_valid_observations is False or \
-                kwargs.get('reread_fields_file', False):
+        if (self.scheduler.has_valid_observations is False or
+                kwargs.get('reread_fields_file', False)):
             self.scheduler.read_field_list()
 
         self.scheduler.get_observation(*args, **kwargs)
```

Comment:
This looks like it should return something. Why not return what get_observation returns?

Reply:
Good idea to make it explicit, but `get_observation` actually also sets `current_observation`, which is then returned. Should maybe think about renaming this.